### ISSUE TITLE
Potential error while running tcpdump, did not expect the following standard error output

### ISSUE THREAD

**Issue by shadoww0** [cuckoosandbox/cuckoo#1584]
I am using Windows 7 as host and guest.
For sniffing I am using windump.exe from https://www.winpcap.org/windump/
I assume cuckoo fails to parse the sniffer's stdout stream. Which tcpdump is recommended for running cuckoo on Windows?

**Comment by doomedraven** [comment]
will you provide any issue?

Posting messages saying just something like “Doesn’t work, help me” are completely useless. If something is not working report the error, paste the logs, the config file, the information on the virtual machine, the results of the troubleshooting, etc. Give context. We are not wizards and we don’t have a crystal ball.

**Comment by shadoww0** [comment]
[cuckoo.conf.txt](https://github.com/cuckoosandbox/cuckoo/files/1015613/cuckoo.conf.txt)
[log.txt](https://github.com/cuckoosandbox/cuckoo/files/1015614/log.txt)
[vmware.conf.txt](https://github.com/cuckoosandbox/cuckoo/files/1015615/vmware.conf.txt)

anything else log you need let me know

**Comment by doomedraven** [comment]
>>I assume cuckoo fails to parse the sniffer's stdout stream.

would be useful if you post what is wrong

**Comment by shadoww0** [comment]
Line 13 in log.txt there is an exception:

[cuckoo.core.plugins] ERROR: Unable to stop auxiliary module: Sniffer

Is it normal??

**Comment by doomedraven** [comment]
no, but there should be more context, as normally this happens when vm fail

**Comment by jbremer** [comment]
You can relax a little bit @doomedraven :-)
The Sniffer module is quite strict as to its expected output and currently hasn't been tested against `windump.exe`, so what you described is most likely accurate; some additional rules have to be created to support it. Reasoning behind this is that in the past actual errors returned by `tcpdump` weren't properly followed up on in Cuckoo, resulting in silent errors towards the users.
As I don't have a Windows/`windump.exe` setup myself currently; could you post the full command-line required to run `windump.exe` (I assume it takes at least some different parameters than `tcpdump`?) as well as the full output based on one or two runs? Thanks!

Note: you could change the `raise CuckooOperationalError(...` on line 128 of `cuckoo/auxiliary/sniffer.py` to a simple `log.info(...)` and get all of the output as Cuckoo sees it.

**Comment by shadoww0** [comment]
In log.txt:

  did not expect the following standard error output: 'C:\\Tools\\windump.exe:listening on \\Device\\NPF_{DC0D91A2-174D-405C-92BF-F4DB8C3E0720}\r'

At the end of the line there is an extra '\r' character. Could this be related to Windows style line endings in the output?

[windump.txt](https://github.com/cuckoosandbox/cuckoo/files/1023856/windump.txt)

**Comment by Yvonne2020219** [comment]
I encounter this problem too. Fixing the problem requires adding some code.
Sniffer.py 
![5241](https://user-images.githubusercontent.com/9027097/84121251-7afe5e00-aa69-11ea-9b78-6816a9135107.png)
